pkg/kubelet/metrics/metrics.go

```diff
@@ -431,6 +445,54 @@ var (
 		},
 		[]string{"container_state"},
 	)
+	// StartedPodsTotal is a counter that tracks pod sandbox creation operations
+	StartedPodsTotal = metrics.NewCounter(
```

Comment:
Note to self: this one is `NewCounter` but the rest of these new metrics are `NewCounterVec` because this one doesn't have any additional labels (e.g. container type, error).